src/core_atmosphere/physics/mpas_atmphys_driver_radiation_lw.F

```diff
@@ -690,34 +730,44 @@ subroutine radiation_lw_to_MPAS(configs,diag_physics,tend_physics,its,ite)
  call mpas_pool_get_array(diag_physics,'lwcf'  ,lwcf  )
  call mpas_pool_get_array(diag_physics,'lwdnb' ,lwdnb )
  call mpas_pool_get_array(diag_physics,'lwdnbc',lwdnbc)
+ call mpas_pool_get_array(diag_physics,'lwdnbcln',lwdnbcln)
```

Comment:
These (733,736,739,742,746) and their respective fills below should be under the IF (calc_clean)